packages/contracts/contracts/optimistic-ethereum/OVM/accounts/OVM_ProxyEOA.sol

```diff
@@ -16,12 +16,21 @@ import { Lib_Bytes32Utils } from "../../libraries/utils/Lib_Bytes32Utils.sol";
  */
 contract OVM_ProxyEOA {
```

Comment:
In general I think we should be inheriting from https://github.com/OpenZeppelin/openzeppelin-contracts/tree/master/contracts/proxy/ERC1967, but not for this PR.